validator/src/lib.rs

```diff
@@ -277,6 +289,14 @@ fn validate_temporal<E: Interpreter + Clone + Display>(
         }
         prev_timepoint = Some(timepoint);
     }
+    if let Some(min_epsilon) = min_epsilon {
+        if env.epsilon < *min_epsilon {
+            bail!(
+                "Found an epsilon equals to {}, the minimal accepted value is {min_epsilon}",
```

Comment:
Should be more explicit: not "epsilon" but "a minimal delay of between ... and ... is not respected"